cmd/cluster/resize/cmd.go

```diff
@@ -74,7 +74,7 @@ func (r *Resize) New() error {
 		return err
 	}
 
-	c, err := k8s.New(cluster.ID(), client.Options{Scheme: scheme})
+	c, err := k8s.NewAsBackplaneClusterAdmin(cluster.ID(), client.Options{Scheme: scheme})
```

Comment:
What do you think about adding a clientAdmin and using it when we need to escalate to backplane-cluster-admin kind of like `hive` and `hiveAdmin`

Reply:
I *think* by removing the taint process we won't need the backplane elevation anymore, hopefully!